### DISCUSSION TITLE
Scala-steward - better options?

### DISCUSSION THREAD

**Discussion by jastice** [build-server-protocol/build-server-protocol#565] [Proposals] [answered]
Scala Steward is nice for keeping the versions up to date, but I find it kind of high overhead for what it is - a notification goes to all the maintainers for every minor version bump, then another when some maintainer merges it. Also we have many commits that are just version bumps, which kind of buries the history of real changes. 

Has somebody come up with an idea to do version bumps more low-key, or in bulk?

**Comment by ckipp01** [comment]
One option could be to use [grouping](https://github.com/scala-steward-org/scala-steward/blob/main/docs/repo-specific-configuration.md) via steward. It was a new feature added not that long ago. I typically don't love grouping or mind the amount of prs, but if this is something that is bothersome to people we can add grouping.

**Comment by jastice** [comment, marked as answer]
Grouping looks good to me. Apparently it's also very recently available [for dependabot](https://github.com/dependabot/dependabot-core/issues/1190). What are your concerns with it?
